docs/lib/push-notifications/fragments/js/getting-started.md

```diff
@@ -206,48 +199,9 @@ Push Notifications category is integrated with [AWS Amplify Analytics category](
 
     The CLI will prompt for your *p12 certificate path*, enter it respectively.
 
-5. Open *ios/myapp.xcodeproj* project file with Xcode.
-
-6. Using Xcode, **manually link** the `PushNotificationIOS` library to your project. Please follow those steps in [React Native developer Documentation](https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking) (Step 3 is not required)
-
-7. Add the following code at the top on the file *AppDelegate.m*:
-
-    ```
-    #import <React/RCTPushNotificationManager.h>
-    ```
-
-8. And then in your `AppDelegate` implementation add the following code:
+5. Setup Xcode for push notification support by following the "Add Capabilities : Background Mode - Remote Notifications" and "Augment AppDelegate" sections found in the: [React Native Push Notifications Documentation](https://github.com/react-native-community/push-notification-ios)
 
-    ```c
-    // Required to register for notifications
-    - (void)application:(UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
-    {
-    [RCTPushNotificationManager didRegisterUserNotificationSettings:notificationSettings];
-    }
-    // Required for the register event.
-    - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken
-    {
-    [RCTPushNotificationManager didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
-    }
-    // Required for the notification event. You must call the completion handler after handling the remote notification.
-    - (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo
-                                                            fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
-    {
-    [RCTPushNotificationManager didReceiveRemoteNotification:userInfo fetchCompletionHandler:completionHandler];
-    }
-    // Required for the registrationError event.
-    - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error
-    {
-    [RCTPushNotificationManager didFailToRegisterForRemoteNotificationsWithError:error];
-    }
-    // Required for the localNotification event.
-    - (void)application:(UIApplication *)application didReceiveLocalNotification:(UILocalNotification *)notification
-    {
-    [RCTPushNotificationManager didReceiveLocalNotification:notification];
-    }
-    ```
-
-9. Update General App settings:
+6. Update General App settings:
```

Comment:
Idle thought: It would be good if we could use `1.` for all ordered lists to make it easier for contributors to make changes without having to update bullet numbers everywhere. Probably outside of the bounds of this change.

Reply:
Markdown modules in some IDE's generally work this out but good idea for a future change